Decompiler/Decompiler.cs

```diff
@@ -420,7 +420,7 @@ private void ProcessFile(string path, Stream stream, string originalPath = null)
 
                 if (OutputFile != null)
                 {
-                    using var contentFile = FileExtract.Extract(resource);
+                    using var contentFile = FileExtract.Extract(resource, null);
```

Comment:
perhaps a future improvement (not in this pr): get a file loader in here (vpk or from disk when parsing raw file).

Reply:
Indeed, can't properly decompile a vmat_c from disk the way it is right now, as it won't find the textures.